dao/src/main/java/org/thingsboard/server/dao/model/ModelConstants.java

```diff
@@ -425,6 +425,8 @@ private ModelConstants() {
     public static final String RULE_CHAIN_FIRST_RULE_NODE_ID_PROPERTY = "first_rule_node_id";
     public static final String RULE_CHAIN_ROOT_PROPERTY = "root";
     public static final String RULE_CHAIN_CONFIGURATION_PROPERTY = "configuration";
+    public static final String RULE_CHAIN_ERROR_PRESENT_PROPERTY = "error_present";
+    public static final String RULE_CHAIN_INFO_VIEW_NAME = "rule_chain_info";
```

Comment:
I don't see any tables rule_chain_info in SQL.
What this constant is needed for?